packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -21,7 +21,53 @@ This package provides a collection of common values that are used in storybook b
 - `options`: Object containing additional context for the input provider. Optional.
   - `defaultService`: String to set the default for the select knob in storybook.
 
-`dirDecorator` - A storybook decorator function that uses `inputProvider` internally to provide direction control. It calls the storybook function with an object containing `dir`, `script` and the `service` name.
+**(Deprecated)** `dirDecorator` - A storybook decorator function that uses `inputProvider` internally to provide direction control. It calls the storybook function with an object containing `dir`, `script` and the c name.
+
+`withServicesKnob` - Is a function that returns a storybook decorator function that adds a `Select a service` dropdown to the knobs panel. When a service is selected from the dropdown it does 2 things:
+
+1. Provides the decorated stories with the following properties that can be passed into components:
+
+- `text`: A string of text in the language of the chosen service.
+- `dir`: The reading directionality of the chosen service e.g. `ltr` or `rtl`
+- `script`: The chosen service's script typography settings e.g. the font-size and line-heights.
+- `service`: The name of the chosen service e.g. `arabic`
+
+2. Toggles the layout directionality of the chosen service.
+
+The `withServicesKnob` function accepts 2 arguments:
+
+- `defaultService`(String): The default selected service of the services dropdown e.g. `arabic`. The default is `news`.
+- `services`(Array): A list of services that the dropdown will display. The default is all services.
+
+An example of `withServicesKnob` usage:
+
+```js
+storiesOf('Components|Paragraph', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob()); // default selected service is `news`
+  .add(
+    'A paragraph with English text',
+    ({ text, script, service }) => (
+      <Paragraph script={script} service={service}>
+        {text}
+      </Paragraph>
+    ),
+```
+
+To set a default service:
+
+```js
+storiesOf('Components|Paragraph', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob('arabic', ['news', 'arabic', 'amharic'])); // default selected service is `arabic` and the available services in the dropdown are `news`, `arabic`, `amharic`
```

Comment:
this argument list might work better as an object - won't be clear from reading code what the args mean, can't pass a service list without setting default service, and will be more problematic if we add more options in the future

Reply:
I was literally this minute thinking that! Cool I'll update this